flowmachine/tests/test_subsetting.py

```diff
@@ -6,251 +6,241 @@
+    """
+    rog = RadiusOfGyration("2016-01-01", "2016-01-02")
+    low = 150
+    high = 155
+    rog_df = get_dataframe(rog).query(
+        "{low} <= rog <= {high}".format(low=low, high=high)
+    )
+    sub = rog.numeric_subset(col="rog", low=low, high=high)
+    sub.store().result()
+    assert sub.is_stored
+    # Test that the store is of the right length
+    sub = rog.numeric_subset(col="rog", low=low, high=high)
+    assert len(get_dataframe(sub)) == len(rog_df)
+
+
+def test_can_subset_with_list_of_subscribers(get_dataframe):
+    """
+    flowmachine.daily_location can be subset with a list of subscribers.
+    """
+    dl = daily_location("2016-01-03")
+    subscriber_list = list(get_dataframe(dl).head(8).subscriber)
+    sub = get_dataframe(dl.subset(col="subscriber", subset=subscriber_list))
+    assert set(subscriber_list) == set(sub.subscriber)
+
+
+def test_can_subset_with_single_subscriber(get_dataframe):
+    """
+    flowmachine.daily_location can be subset with a single subscriber.
+    """
+    dl = daily_location("2016-01-03")
+    single_subscriber = list(get_dataframe(dl).head(8).subscriber)[3]
+    sub = get_dataframe(dl.subset(col="subscriber", subset=single_subscriber))
+    assert set(sub.subscriber) == {single_subscriber}
+
+
+def test_can_subset_with_list_containing_single_subscriber(get_dataframe):
+    """
+    flowmachine.daily_location can be subset with a list containing a single subscriber.
+    """
+    dl = daily_location("2016-01-03")
+    single_subscriber = list(get_dataframe(dl).head(8).subscriber)[3]
+    sub = get_dataframe(dl.subset(col="subscriber", subset=[single_subscriber]))
+    assert set(sub.subscriber) == {single_subscriber}
+
+
+def test_can_getitem_location(get_dataframe):
+    """
+    flowmachine.daily_location can use get_item
+    """
+    dl = daily_location("2016-01-03")
+    single_subscriber = list(get_dataframe(dl).head(8).subscriber)[3]
+    sub = get_dataframe(dl[single_subscriber])
+    assert set(sub.subscriber) == {single_subscriber}
+
+
+def test_special_chars(get_dataframe):
+    """Special characters don't break subsets"""
+    dl = daily_location("2016-01-03")
+    sub = dl.subset(col="subscriber", subset=["Murray'"])
+    get_dataframe(sub)
+    sub = dl.subset(col="subscriber", subset=["Murray'", "Horace"])
+    get_dataframe(sub)
+    sub = dl.subset(col="subscriber", subset="Murray'")
+    get_dataframe(sub)
+    sub = dl.subset(col="subscriber", subset="Murray'")
+    get_dataframe(sub)
+
+
+def test_can_get_item_subscriber_metric(get_dataframe):
+    """
+    flowmachine.SubscriberFeature allows for getting items
+    """
+    rog = RadiusOfGyration("2016-01-01", "2016-01-03")
+    dl = daily_location("2016-01-03")
+    single_subscriber = list(get_dataframe(dl).head(8).subscriber)[3]
+    sub = get_dataframe(rog[single_subscriber])
+    assert set(sub.subscriber) == {single_subscriber}
+
+
+def test_calling_parent_does_break_subsetting(get_dataframe):
```

Comment:
Hmm, I'm not sure I see how this test checks that subsetting is _broken_. Doesn't it check that it works, or am I misinterpreting the test?

Reply:
I do believe there should be a not in there now you mention it!